### DISCUSSION TITLE
Occurrences module - How to change status of participants in mass?

### DISCUSSION THREAD

**Discussion by LaurensCleyman** [YetiForceCompany/YetiForceCRM#16309] [How to]
Hi all, 

In the occurrences module, we have the option to add multiple participants.
We can change the status of participants one by one by clicking the yellow icon:

![image](https://user-images.githubusercontent.com/16014524/181827749-8ab88d99-388d-47f4-9798-853cc99b0709.png)
![image](https://user-images.githubusercontent.com/16014524/181827997-6faa68d5-f861-4a25-b869-2852910b3a83.png)

**My question:**
How can we change the status of multiple participants in mass? 

The Actions menu does not give the mass edit option:

![image](https://user-images.githubusercontent.com/16014524/181827842-c03844ab-47ee-4f77-8f0d-9875f81e3230.png)

Kind regards,

Laurens

**Comment by apcloic** [comment]
@LaurensCleyman 
Occurrences status and comment fields are not standard fields, like they are also not available in PDF module.
#13486
I believe you have to create a custom function.
